deeplearning4j-scaleout/deeplearning4j-scaleout-parallelwrapper/src/main/java/org/deeplearning4j/parallelism/trainer/DefaultTrainer.java

```diff
@@ -165,6 +165,8 @@ protected void setupIfNeccessary() {
             shouldUpdate = new AtomicBoolean(false);
         if (isStopped == null)
             isStopped = new AtomicBoolean(false);
+        if (lastEtlTime == null)
```

Comment:
Unfortunately no. That volatile change in comment above was exactly about this field being null.  Lombok issue in other words.